firefox-ios/firefox-ios-tests/Tests/ClientTests/TabTests.swift

```diff
@@ -202,6 +209,128 @@ class TabTests: XCTestCase {
         XCTAssertTrue(normalInctiveTab1.isSameTypeAs(normalInctiveTab2))
         XCTAssertTrue(normalInctiveTab2.isSameTypeAs(normalInctiveTab1))
     }
+
+    // MARK: - Document Handling
+
+    func testEnqueueDocument() {
+        let subject = Tab(profile: mockProfile, windowUUID: windowUUID)
```

Comment:
I know the other tests in this test class isn't using it, but I think having `trackForMemoryLeaks` for the `TabTests` would be worthwhile. We had some leaks related to tabs and TabWebView not so long ago (and still have some issues even I think).